src/main/java/us/bmark/android/BookmarkListsActivity.java

```diff
@@ -59,11 +68,21 @@ public void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.main);
         pager = (ViewPager) findViewById(R.id.pager);
         createFragments(savedInstanceState);
-
-        final ActionBar actionBar = getActionBar();
+        id = getIntent().getStringExtra("caller");
+        //Get Sharedpreferences
+        settings = getSharedPreferences("mypref", MODE_PRIVATE);
+        //Get editor
+        editor = settings.edit();
+        //always set col to the color saved last time user had chosen it(for the first time 1--dark)
+        col = settings.getString("mycolor", "1");
```

Comment:
I'm wondering if it won't be clearer to the color or theme names as string constants instead of ```"1"```, ```"2"``` etc